charts/portal/values.yaml

```diff
@@ -67,3 +67,22 @@ validation:
   path: /validate
   service: "extension-manager-operator-server"
   protocol: "http"
+
+gatewayApi:
+  # -- toggle to enable the Gateway API
+  enabled: false
+  # -- configuration for the HTTPRoute resource
```

Comment:
Please avoid comments on such parent nodes, as it leads to complex objects being generated in the readme.